end_to_end/test_generate_param_only_checkpoint.sh

```diff
@@ -20,7 +20,7 @@ dry_run=false
 run_id=test_model_0b_$(date +%Y-%m-%d-%H)
 dataset_path=gs://test-maxtext-dataset
 base_output_directory=gs://test-maxtext-output
-ici_tensor_parallelism=8
+ici_tensor_parallelism=4
```

Comment:
Why did this change happen?

Reply:
I was testing it locally on v4-8, so it was failing with 8. Will change back to 8